2022-01-31 22:36:06,390 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:36:06,431 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:36:06,431 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:37:14,111 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57421)
2022-01-31 22:37:14,622 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643657834.111168, 'user': {'account_name': 'Guest'}}
2022-01-31 22:37:15,127 INFO server.py Клиент ('192.168.0.49', 57421) отключился от сервера.
2022-01-31 22:40:32,505 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57434)
2022-01-31 22:40:32,506 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658032.505548, 'user': {'account_name': 'Guest'}}
2022-01-31 22:43:11,759 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57453)
2022-01-31 22:43:13,273 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658192.347498, 'user': {'account_name': 'Guest'}}
2022-01-31 22:46:30,905 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57467)
2022-01-31 22:46:39,471 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658394.685927, 'user': {'account_name': 'Guest'}}
2022-01-31 22:49:10,410 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57475)
2022-01-31 22:49:18,470 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658551.9511738, 'user': {'account_name': 'TestSender2'}}
2022-01-31 22:49:38,606 INFO server.py Клиент ('192.168.0.49', 57475) отключился от сервера.
2022-01-31 22:52:32,934 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:52:32,949 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:52:32,950 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:52:42,262 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:52:42,292 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:52:42,292 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:53:18,509 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:53:18,553 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:53:18,554 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:53:20,736 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:53:20,762 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:53:20,762 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:53:33,633 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57497)
2022-01-31 22:53:34,144 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658813.6329489, 'user': {'account_name': 'TestSender2'}}
2022-01-31 22:53:34,648 INFO server.py Клиент ('192.168.0.49', 57497) отключился от сервера.
2022-01-31 22:54:36,365 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:54:36,386 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:54:36,386 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:54:43,470 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57514)
2022-01-31 22:54:43,972 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658883.470144, 'user': {'account_name': 'TestSender2'}}
2022-01-31 22:54:44,477 INFO server.py Клиент ('192.168.0.49', 57514) отключился от сервера.
2022-01-31 22:55:57,813 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57520)
2022-01-31 22:55:57,813 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643658957.8129122, 'user': {'account_name': 'TestSender2'}}
2022-01-31 22:57:17,280 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57527)
2022-01-31 22:57:17,785 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659037.280428, 'user': {'account_name': 'TestSender2'}}
2022-01-31 22:58:19,107 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57536)
2022-01-31 22:58:19,613 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659099.107488, 'user': {'account_name': 'TestSender1'}}
2022-01-31 22:58:43,613 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:58:43,675 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:58:43,675 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:59:51,348 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 22:59:51,356 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 22:59:51,357 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 22:59:59,763 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57564)
2022-01-31 23:00:00,268 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659199.7630668, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:00:06,478 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57567)
2022-01-31 23:00:06,478 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659206.478135, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:00:58,341 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57576)
2022-01-31 23:00:58,342 INFO server.py Клиент ('192.168.0.49', 57567) отключился от сервера.
2022-01-31 23:00:58,847 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659258.3413708, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:01:03,846 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57577)
2022-01-31 23:01:03,847 INFO server.py Клиент ('192.168.0.49', 57564) отключился от сервера.
2022-01-31 23:01:04,351 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659263.846829, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:02:30,387 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57591)
2022-01-31 23:02:30,892 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659350.3872871, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:04:02,445 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57603)
2022-01-31 23:04:03,450 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659442.91594, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:08:04,367 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:08:04,402 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:08:04,402 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:08:08,728 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57625)
2022-01-31 23:08:09,240 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659688.7275252, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:08:22,330 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643659701.9548512, 'from': 'TestSender1', 'to': 'TestSender2', 'mess_text': 'dasfa'}
2022-01-31 23:08:22,330 ERROR server.py Пользователь TestSender2 не зарегистрирован на сервере, отправка сообщения невозможна.
2022-01-31 23:09:18,931 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57637)
2022-01-31 23:09:19,436 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659758.9315588, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:09:39,078 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643659778.898027, 'from': 'TestSender2', 'to': 'TestSender1', 'mess_text': 'fgjghdjghj'}
2022-01-31 23:09:39,086 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender2.
2022-01-31 23:10:03,272 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643659803.241503, 'from': 'TestSender1', 'to': 'TestSender2', 'mess_text': 'dsadasd'}
2022-01-31 23:10:03,272 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender1.
2022-01-31 23:12:24,289 INFO server.py Клиент ('192.168.0.49', 57637) отключился от сервера.
2022-01-31 23:12:24,589 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57650)
2022-01-31 23:12:25,091 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659944.5894341, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:12:36,169 INFO server.py Клиент ('192.168.0.49', 57625) отключился от сервера.
2022-01-31 23:12:36,335 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57652)
2022-01-31 23:12:36,335 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659956.3351748, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:12:57,180 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:12:57,206 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:12:57,207 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:13:10,072 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:13:10,130 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:13:10,130 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:13:15,178 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57656)
2022-01-31 23:13:15,681 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659995.178615, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:13:19,261 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57657)
2022-01-31 23:13:19,262 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643659999.261207, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:13:50,010 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660029.539308, 'from': 'TestSender2', 'to': 'TestSender2', 'mess_text': 'sgfdhdf'}
2022-01-31 23:13:50,010 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender2.
2022-01-31 23:14:17,688 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660057.294275, 'from': 'TestSender2', 'to': 'TestSender1', 'mess_text': 'asfds'}
2022-01-31 23:14:17,688 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender2.
2022-01-31 23:14:42,856 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660082.578159, 'from': 'TestSender2', 'to': 'TestSender2', 'mess_text': 'wafsdbgfdg'}
2022-01-31 23:14:42,864 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender2.
2022-01-31 23:15:16,217 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:15:16,235 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:15:16,236 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:15:18,923 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:15:18,950 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:15:18,950 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:15:21,655 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:15:21,698 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:15:21,698 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:16:07,156 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:16:07,212 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:16:07,212 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:16:14,700 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57689)
2022-01-31 23:16:15,204 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643660174.700465, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:16:24,774 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57691)
2022-01-31 23:16:24,775 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643660184.774424, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:16:42,901 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660202.7805362, 'from': 'TestSender1', 'to': 'TestSender1', 'mess_text': 'asfdasfdsgdfsnjghku'}
2022-01-31 23:16:42,909 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender1.
2022-01-31 23:17:00,532 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660220.1967292, 'from': 'TestSender1', 'to': 'TestSender2', 'mess_text': 'sdgfklsnlvgknw'}
2022-01-31 23:17:00,533 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender1.
2022-01-31 23:17:15,131 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660234.922261, 'from': 'TestSender2', 'to': 'TestSender1', 'mess_text': 'sdvsdv'}
2022-01-31 23:17:15,131 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender2.
2022-01-31 23:18:02,479 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/10-cliens_server_applications/client-server-app/Lesson_8/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.49'] функции ServerApp().
2022-01-31 23:18:02,507 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.49. Если адрес не указан, принимаются соединения с любых адресов.
2022-01-31 23:18:02,508 DEBUG server.py Сервер запущен. Адрес: 192.168.0.49 Порт: 8888
2022-01-31 23:18:08,234 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57697)
2022-01-31 23:18:08,737 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643660288.236982, 'user': {'account_name': 'TestSender1'}}
2022-01-31 23:18:12,627 INFO server.py Установлено соедение с Клиентом: ('192.168.0.49', 57698)
2022-01-31 23:18:13,132 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1643660292.627532, 'user': {'account_name': 'TestSender2'}}
2022-01-31 23:18:35,292 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660315.068738, 'from': 'TestSender1', 'to': 'TestSender1', 'mess_text': 'sadafd'}
2022-01-31 23:18:35,293 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender1.
2022-01-31 23:18:57,956 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660337.855246, 'from': 'TestSender2', 'to': 'fafasda', 'mess_text': ''}
2022-01-31 23:18:57,957 ERROR server.py Пользователь fafasda не зарегистрирован на сервере, отправка сообщения невозможна.
2022-01-31 23:19:12,567 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660352.522114, 'from': 'TestSender2', 'to': 'TestSender1', 'mess_text': 'dasfdasf'}
2022-01-31 23:19:12,567 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender2.
2022-01-31 23:19:21,116 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1643660360.627274, 'from': 'TestSender2', 'to': 'TestSender2', 'mess_text': 'fsdfvdsvgfd'}
2022-01-31 23:19:21,116 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender2.
2022-01-31 23:19:32,703 INFO server.py Клиент ('192.168.0.49', 57698) отключился от сервера.
2022-01-31 23:19:46,302 INFO server.py Клиент ('192.168.0.49', 57697) отключился от сервера.
